Description: yockgen 01 test patch
 <long description that can span multiple lines, optional>
Author: Mah Yock Gen
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: 2021-08-16 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/hugin1/hugin/huginApp.cpp
+++ b/src/hugin1/hugin/huginApp.cpp
@@ -1,5 +1,5 @@
 // -*- c-basic-offset: 4 -*-
-
+//yockgen added somethig here...bla bla bla for testing
 /** @file huginApp.cpp
  *
  *  @brief implementation of huginApp Class
